qutip/core/qobj.py

```diff
@@ -1733,14 +1728,37 @@ def isunitary(self):
         return self._isunitary
 
     @property
-    def shape(self): return self.data.shape
-
-    isbra = property(isbra)
-    isket = property(isket)
-    isoper = property(isoper)
-    issuper = property(issuper)
-    isoperbra = property(isoperbra)
-    isoperket = property(isoperket)
+    def shape(self): return self._data.shape
```

Comment:
```suggestion
    def shape(self):
        """Return the shape of the Qobj data."""
        return self._data.shape
```
Might as well make this formatting nice while we're here.